lib/utils.js

```diff
@@ -330,6 +331,22 @@ const utils = module.exports = {
         let relativePath = path.relative(root, link).replace(/\\/g, '/');
 
         return '/' + relativePath;
-    }
+    },
+
+
+    /**
+     * Load a set of files
+     * @param  {string|array} dir
+     * @param  {string}       pattern
+     * @return {array}
+     */
+    loadFiles(dir, pattern) {
```

Comment:
Can you add a test for this utility in test/1.utils.js?

You can run tests for just that one file by running `npx mocha test/1.utils.js` from the root.